R/import_data.R

```diff
@@ -26,16 +26,15 @@ isColumnTypesCorrect <- function(col.types){
 
 #' Checks if the complete file was read or not.
 #' 
-#' \code{isFullFile} returns a logical scalar after checking 
+#' \code{isPreview} returns a logical scalar after checking 
 #' the attributes of the dataframe sent as an argument. 
-#' Returns TRUE if it is the complete file. Returns FALSE if 
-#' it is a partial file read. 
+#' Returns TRUE if it was read as a preview. 
 #' 
 #' @param user.data.frame the dataframe returned after reading the file
 #' @return A logical scalar
-isFullFile <- function(user.data.frame){
+isPreview <- function(user.data.frame){
   
-  if (attr(user.data.frame, "full.file") == TRUE){
+  if (attr(user.data.frame, "preview") == TRUE){
```

Comment:
You  __add__ the `preview` attribute if and only if the returned structure __is__ a preview. Otherwise you need to add the preview argument to every method (`iNZread.dta` etc) even though it is redundant (the function only returns the full file). This ties in to the other comment that you can delete the attribute lines completely (if there isn't a `preview` argument, then it's the full file).